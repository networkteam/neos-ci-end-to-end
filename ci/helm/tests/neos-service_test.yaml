templates:
  - neos-service.yaml
tests:
  - it: matches snapshot with review deployment
    set:
      default:
        annotations:
          app.gitlab.com/env: foo
          app.gitlab.com/app: bar
    release:
      name: my-release
    asserts:
      - matchSnapshot: {}
