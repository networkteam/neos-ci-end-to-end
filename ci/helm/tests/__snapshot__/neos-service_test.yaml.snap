matches snapshot with review deployment:
  1: |
    apiVersion: v1
    kind: Service
    metadata:
      annotations:
        app.gitlab.com/app: bar
        app.gitlab.com/env: foo
      labels:
        app.kubernetes.io/component: neos
        app.kubernetes.io/instance: my-release
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: neos-ci
        app.kubernetes.io/version: 1.0.0
        helm.sh/chart: neos-ci-0.1.0
      name: my-release-neos
    spec:
      ports:
      - name: http
        port: 80
        protocol: TCP
        targetPort: http
      selector:
        app.kubernetes.io/component: neos
        app.kubernetes.io/instance: my-release
        app.kubernetes.io/name: neos-ci
      type: ClusterIP
